ResGen/ResGen.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Maybe consider adding this project to the main solution file (sln)?

Reply:
since I don't use VS at all, I'm not quite sure what this would actually look like. Would you be willing to do that after this is merged?

Reply:
Ok, I can do that. There might be a later PR from me to try to improve the VS experience (without affecting the VSCode experience). What is really nice with VS it that when I build in debug, then I can just go to the net451 bin folder of the rules and easily attach the debugger to it with `ipmo .\Microsoft.Windows.PowerShe
ll.ScriptAnalyzer.dll; Invoke-sccriptanalyzer -scriptdefinition '$foo=1' -AttachAndDebug`